packages/terra-framework-docs/src/terra-dev-site/doc/slider/Examples.2/SliderWithValueUpdate.5.doc.mdx

```diff
@@ -1,3 +1,9 @@
 import SliderWithValueUpdate from './SliderWithValueUpdate?dev-site-example';
 
+# Slider with Value Update
+
+### Description
+
+This example demonstrates a [Slider](/components/cerner-terra-framework-docs/slider/about) with value updates using the 'value' prop and 'onChange' prop.
```

Comment:
I think this and the 'onChange' example could be merged.

Reply:
@eawww I remember there was request to have an additional example with programmatic value updates from dev review on my actual Slider component implementation PR. I also think both there examples show different ways of code implementation for slider. 
I have changed the wording on the programmatically setting value example to specifically try and differentiate between these examples. Let me know if that is fine.